extensions/fhir/runtime/pom.xml

```diff
@@ -74,6 +70,11 @@
             <groupId>org.jboss.logging</groupId>
             <artifactId>commons-logging-jboss-logging</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>com.oracle.substratevm</groupId>
+            <artifactId>svm</artifactId>
```

Comment:
Is this really needed in FHIR? I see no substitutions or anything that would require this dep.

Reply:
> I see no substitutions

I see them now, sorry. Had to hit F5 in Eclipse.